internal/attestation/certcache/cached_client.go

```diff
@@ -59,27 +60,27 @@ func (c *CachedHTTPSGetter) GetContext(ctx context.Context, url string) (map[str
 	default:
 	}
 
-	if snpCrlURL.MatchString(url) || tdxCrlURL.MatchString(url) {
-		// For CRLs always query. When request failure, fallback to cache.
-		c.logger.Debug("Request CRL", "url", url)
+	if snpCrlURL.MatchString(url) || tdxRootCrlURL.MatchString(url) || tdxBaseURL.MatchString(url) {
+		// For CRLs or TDX TCB/QeIdentity always query. When request failure, fallback to cache.
+		c.logger.Debug("Request URL", "url", url)
 		header, body, err := c.ContextHTTPSGetter.GetContext(ctx, url)
 		if err == nil {
 			if data, err := json.Marshal(cacheEntry{header, body}); err == nil {
 				c.cache.Set(url, data)
 			} else {
-				c.logger.Warn("Failed to marshal CRL response, not writing to cache", "error", err)
+				c.logger.Warn("Failed to marshal response, not writing to cache", "error", err)
 			}
 			return header, body, nil
 		}
-		c.logger.Warn("Failed requesting CRL from KDS/PCS", "error", err)
+		c.logger.Warn("Failed requesting URL from KDS/PCS", "error", err)
```

Comment:
At this point, it would probably make sense to log the failing URL, too. Preferably as `log := c.logger.With("url", url)` at the beginning of the function, then you could remove a bunch of log arguments.